src/openms/include/OpenMS/CHEMISTRY/EmpiricalFormula.h

```diff
@@ -161,12 +170,21 @@ namespace OpenMS
       The details of the calculation of the isotope distribution
       are described in the doc to the IsotopeDistribution class.
 
-      @param max_depth: the maximum isotope which is considered, if 0 all are reported
+      @param method: the maximum isotope which is considered, if 0 all are reported
     */
-    IsotopeDistribution getIsotopeDistribution(UInt max_depth) const;
+    IsotopeDistribution getIsotopeDistribution(const IsotopePatternGenerator& method) const;
+
 
     /**
-      @brief returns the fragment isotope distribution of this given a precursor formula
+       @brief formula's isotope distribution using the CoarseIsotopeDistribution
+       This is used as a wrapper for the pyOpenMS
+     */
+    IsotopeDistribution getIsotopeDistribution() const;
```

Comment:
I dont think this is needed (see pyOpenMS wrapper and tests)

Reply:
Yes since you suggested the stub call with the coarse method, that works great btw :+1: